### ISSUE TITLE
Parsing arrays that contain '[ xxx]' strings

### ISSUE THREAD

**Issue by YaphetKG** [RedisGraph/redisgraph-bulk-loader#45]
When trying to bulk load data like this :
```
id:ID|names:ARRAY
bogusID|['selenophosphoric acid','trihydroxidoselenidophosphorus','[P(OH)3Se]']
```
This raises an exception where bulk loader tries to infertypes with in the array elements and ends up trying to parse 
`[P(OH)3Se]` as an array. Is it relevant to try to parse array internal items as arrays ? Not sure if redisgraph supports nested arrays?

**Comment by jeffreylovitz** [comment]
Hi @YaphetKG,

RedisGraph and the bulk loader both support nested arrays, but since the schema header doesn't specify the types of values within array fields, we must resort to inference.

I believe this problem is resolved by your PR, correct?

**Comment by YaphetKG** [comment]
Hi @jeffreylovitz, that  is correct.  Closing this issue. 
Resolved by #46
